### ISSUE TITLE
Tool crashes with a fatal exception

### ISSUE THREAD

**Issue by xtbmedia** [digitalcoyote/NuGetDefense#132]
**Describe the bug**
Nuget defesne tool crashes with a fatal exception
```
Error : Encountered a fatal exception while load settings. Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at NuGetDefense.Scanner.LoadSettings(ScanOptions options) in /home/codingcoyote/Code/NuGetDefense/Src/NuGetDefense.Lib/Scanner.cs:line 107
```

**To Reproduce**
1. Launch the nuget defense tool but do not specify a settings file on the command line

**Expected behavior**
If no settings file is specified on the command line, the default (nugetdefnse.json) from the same folder as the project / solution should be used.

**Screenshots**
![image](https://user-images.githubusercontent.com/48799818/227499902-f1f3bf7d-98b7-4d0f-b0da-9b9395f8dc24.png)

**Tools (please complete the following information):**
 - First encountered in azure devops pipeline, reproducible using in debugger with VS2022 (17.5)
 - OS: Windows 11 (22H2, 22621.1413)

**Comment by digitalcoyote** [comment]
Thank you, this is why I normally don't like to publish low on sleep... I'll try to pull this in tonight.

**Comment by digitalcoyote** [comment]
Released in v3.1.1. Thanks again
